### PR TITLE
Adding vagrant resource files for WSO2 - EI

### PR DESCRIPTION
**Pull request by ctienshi**

### CODE REVIEW

**Review by imesh — COMMENTED** [review]
Great work Ching! Is there any reason to have a folder called "boxes" inside "vagrant-boxes" folder? 
vagrant-boxes/boxes/

**Review by imesh — COMMENTED** [review]
It might be better to use the same convention when naming folders. As I see vagrantFiles folder has followed camel notation while others have used dash as the word separator. For an instance:
- vagrantFiles
- vagrant-boxes

**Comment by ctienshi** [comment]
@imesh It is possible to remove the  "boxes" folder. I will make the necessary changes. (removing the boxes and renaming the folders with the same notation)

**Comment by ctienshi** [comment]
@imesh The requested changes are done and committed in the PR.

File: boxes/base/provisioner/provisioner.sh [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+if [ "$?" -eq "0" ];
+then
+  echo "Successfully copied the org.wso2.securevault-1.0.0-wso2v2 to the server pack."
+else
+  echo "Failed to copy the org.wso2.securevault-1.0.0-wso2v2 to the server pack."
+fi
+
+
+
+
```
Comment (VimukthiPerera) [inline boxes/base/provisioner/provisioner.sh]: Unnecessary space here.

**Review by chamithkumarage — APPROVED** [review]

**Pull request merged**